cpp/perspective/CMakeLists.txt

```diff
@@ -642,8 +651,9 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
 
 		target_link_libraries(psp tbb)
 		target_link_libraries(binding tbb)
-
 		target_link_libraries(binding psp)
+		target_compile_options(psp PRIVATE -Wno-deprecated-register)
```

Comment:
what are these for?

Reply:
Silences python2 deprecation warnings which make our build logs un-readable.

Reply:
You're doing this on windows too though, are you sure this flag will work there?

Reply:
I don't know ... our windows azure builds do not work ...